### ISSUE TITLE
Transfer Learning?

### ISSUE THREAD

**Issue by ChauncyFr** [hzxie/Pix2Vox#15]
How to continue training on the weight of the training you provide, that is, how to perform transfer learning?

**Comment by hzxie** [comment]
Just load the pretrained model and fine tune it on another dataset.

**Comment by ChauncyFr** [comment]
> Just load the pretrained model and fine tune it on another dataset.

python runner.py --weights=./pretrained/Pix2Vox-A-ShapeNet.pth ?

**Comment by hzxie** [comment]
Yes.

**Comment by ChauncyFr** [comment]
> Yes.

I made very few data sets (the data sets are all regular cubes, only 70), and then I tried to use transfer learning to see if the network is valid for my data set. I have found that the loss value in the Test output in each Epoch has not changed. What is going on? Looking forward to your reply!

![](https://tva1.sinaimg.cn/large/006y8mN6ly1g9dqov1lg1j31ti0p4q63.jpg)
